docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -1,57 +1,48 @@
 ---
-description: Configure the static analysis tools and code patterns that Codacy uses to analyze your repositories to ensure that the analysis results are adapted to your scenario.
+description: Configure the static analysis tools and code patterns that Codacy uses to analyze your repositories to ensure that the analysis results fit your needs.
 ---
 
 # Configuring code patterns
 
-By default, Codacy uses a subset of the supported static analysis tools and code patterns to analyze your repositories. These default settings result from community feedback or existing coding standards. However, you can adapt the default settings to your scenario by configuring the tools and code patterns that Codacy uses to analyze your repository.
+{% include-markdown "../assets/includes/admin-access-control-info.md" %}
 
-!!! note
-    -   Organization admins can [change who is allowed to configure code patterns](../organizations/roles-and-permissions-for-organizations.md#change-analysis-configuration).
+By default, Codacy analyzes your repositories using a subset of the supported analysis tools and code patterns. These defaults are based on current best practices and community feedback, and you can adapt them to your needs in several ways:
+
+-   [Configuring tools and code patterns using the Codacy UI](#configuring-tools-and-code-patterns-using-the-codacy-ui)
+-   [Importing configurations from another repository](#import-patterns)
+-   [Using tool configuration files](#using-your-own-tool-configuration-files)
 
-    -   If your repository is following an [organization coding standard](../organizations/using-coding-standards.md), changes made to any tool or code pattern cause the repository to stop following the coding standard. In this case Codacy asks for your confirmation before accepting the changes, and then copies the coding standard configurations to your repository so you can customize them.
+## Configuring tools and code patterns using the Codacy UI
 
-To configure the tools and code patterns for your repository:
+!!! note
+    If you update the configurations of a repository that follows a [coding standard](../organizations/using-coding-standards.md), Codacy copies the coding standard configurations to the repository and the repository stops following the coding standard. You can then customize the repository configurations without affecting the coding standard.
+
+To configure the tools and code patterns for a repository using the Codacy UI:
 
 1.  Open your repository **Code patterns** page.
 
     ![Code patterns page](images/code-patterns.png)
 
-1.  Toggle the tools that Codacy will use to analyze your repository.
+1.  Enable or disable the tools that Codacy should use to analyze the repository.
```

Comment:
> "Should" implies that it will, but it may not if it never has to (if it never finds an issue) 

But it will use the tool even if it doesn't detect an issue, right? 😄 
I find it more clear using "will". 🤓

Reply:
```suggestion
1.  Enable or disable the tools that Codacy will use to analyze the repository.
```